src/main/steps/applicant1/hub-page/sole/content.ts

```diff
@@ -67,6 +67,23 @@ const en = ({ isDivorce, partner, userCase }: CommonContent) => ({
       isDivorce ? 'divorce application' : 'application to end your civil partnership'
     }. <a class="govuk-link" href="https://www.gov.uk/money-property-when-relationship-ends" target="_blank">Find out about dividing money and property</a>`,
   },
+  holdingAndDeemedOrDispensedAccepted: {
```

Comment:
again, would it be possible to move the anchor tag to the template, and pass href/download to it through content